deploy/webhooks/agentinstalladmission_rbac_role.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  annotations:
+  name: system:openshift:assisted-installer:agentinstalladmission
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - configmaps
```

Comment:
What does it do with configmaps?

Reply:
This is needed to access the `extension-apiserver-authentication` configmap